Server/mods/deathmatch/logic/CPlayer.cpp

```diff
@@ -237,14 +229,14 @@ bool CPlayer::IsSubscribed(CElement* pElement, const std::string& strName) const
 
 const char* CPlayer::GetSourceIP()
 {
-    if (m_strIP.empty())
+    if (m_IP.Empty())
```

Comment:
I'd invert the logic here to reduce indentation

Reply:
Less code this way.